docs/usability/ipfs-webui-useabilty-test.md

```diff
@@ -0,0 +1,82 @@
+# IPFS Web UI Usability Testing
+
+The app should make the following actions **easy** to complete
+
+- Add a file and share it with a friend
+- See the files in your repository
```

Comment:
"Files"? Do we mean MFS or all objects in IPFS repo?
It would be much easier if we had better language to talk about those things.
MFS is a really bad acronym and even worse full name (civilians just shut down). 

Digression:
Any thoughts on UX change to use "IPFS Drive" instead of "MFS" (https://github.com/ipfs/specs/issues/174#issuecomment-412912829) across the stack, docs etc? It is a so much better name.

Reply:
We could at least test that language in the app, if it appears there. Does it appear in the app? It would be great to see first if that makes more sense to people before changing everything across all our docs. (I'm sure it will be clearer than MFS, but I wonder if through testing we'd come up with something even better!)

Would 'IPFS Drive' replace.....repository? No, that's not quite it. Hmm. Do you think we're at a place where we can talk at Team Week about how lots of our concepts/terms fit together? ie, to a Normal Human, what `ipfs add` means, what `ipfs get` means, how that relates to, say, an IPFS Drive.....??? It would be great to start to align on an illustration or metaphor for all this that the team could use since often when you change one word, you find out later that it doesn't make sense once *other* terms are changed. Might be asking too much at this stage of development tho :)

Reply:
@lidel I used files here as I don't think that user will typically know there is a difference between "things in the MFS/Drive" and "things in the repo". I think the name "Drive" is better than "MFS", and I think better still would be for all files in your repo to be in you MFS and we just refer to them as "your files".

@meiqimichelle Strong yes from me about making time in team week to figure this out! We have to find a story and a set of terms that work. Not having it slowing us down.

The biggest ones that are causing me trouble today are
- Files in repo vs Files in MFS / Drive. Right now, the list of files in your MFS are a subset of all the files in your repo. We could lobby for that to not be the case in the future, but in the mean time we have to deal with it.

- Upgrade "pinning" to co-hosting. We need to start pushing for apis that let us build apps that let people co-host content and set a duration (1week, 1month, until i need space, forever, kind of thing.) The concept of pinning doesn't seem to work for everyone, and it's an all or nothing thing; If I pin it, it's forever until I unpin it. If I don't pin it, I share it anyway until my repo is full, at which point it may get deleted.